back-end/Models/comments/comment.js

```diff
@@ -0,0 +1,41 @@
+const connection = require('../../db');
+require('dotenv').config();
+
+//Add new comment
+const addComment=(req,res)=>{
+  let sql = "INSERT INTO comment SET ?";
+  let comment = req.body.comment;
+  connection.query(sql, comment,(err, results) => {
+    if(err) throw err;
+    res.json( results);
+  });
+}
+
+//Update comment
+const updateCommentById=(req,res)=>{
+  let comment = req.body.comment
+  let commentID = req.params.comment_id
+  let sql = `UPDATE comment SET comment=? WHERE comment_id=?`;

```

Comment:
same as above

Where there is no validation on who is updating the comment, this means you can update the comments for other users